### DISCUSSION TITLE
使用latex时中文字符报错

### DISCUSSION THREAD

**Discussion by shaoxiang-zheng** [nuaatug/nuaathesis#66] [Q&A]
您好，初学tex，发现编译的时候global.text中 中文的keywords报错 “Package inputenc Error:  Unicode character not set up for use with Latex"，请问是什么原因？感谢！

**Comment by yzwduck** [comment]
可能是用 XeLaTeX 以外的程序来编译了，但我无法根据这一行错误信息来推断。
如需进一步分析问题，请提供用来复现问题的所有代码，并附上编译时生成的 log 文件。

**Comment by shaoxiang-zheng** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    好的！我发现是编译方式不对，感谢您的回复！
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                zhengshaoxiang
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    在2021年9月26日 ***@***.***> 写道： 




可能是用 XeLaTeX 以外的程序来编译了，但我无法根据这一行错误信息来推断。

如需进一步分析问题，请提供用来复现问题的所有代码，并附上编译时生成的 log 文件。



—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
